utils/docker/base/Dockerfile

```diff
@@ -47,3 +48,13 @@ ENV SGX_SDK=/opt/intel/sgxsdk
 ENV PATH=$PATH:$SGX_SDK/bin:$SGX_SDK/bin/x64
 ENV PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$SGX_SDK/pkgconfig
 ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$SGX_SDK/sdk_libs
+
+# install custom protoc
+ARG PROTO_VERSION=3.11.4
+ARG PROTO_ZIP=protoc-${PROTO_VERSION}-linux-x86_64.zip
+ARG PROTO_REPO=https://github.com/google/protobuf/releases/download
+ARG PROTO_DIR=/usr/local/protoc3
```

Comment:
Matter of taste but i think it's nicer to have all build-arg together at the beginning of a docker-file ....